lib/v2/tencent/news/author.js

```diff
@@ -0,0 +1,44 @@
+const got = require('@/utils/got');
+const cheerio = require('cheerio');
+const { parseDate } = require('@/utils/parse-date');
+const timezone = require('@/utils/timezone');
+
+module.exports = async (ctx) => {
+    const mid = ctx.params.mid;
+    const response = await got(`https://pacaio.match.qq.com/om/mediaArticles?mid=${mid}&num=10&page=0`);
+    const reponse = response.data;
+    const title = reponse.mediainfo.name;
+    const description = reponse.mediainfo.intro;
+    const list = reponse.data;
+
+    const items = await Promise.all(
+        list.map((item) => {
+            const title = item.title;
+            const pubDate = timezone(parseDate(item.time), +8);
+            const itemUrl = item.vurl;
+            const author = item.source;
+            const abstract = item.abstract;
+
+            return ctx.cache.tryGet(itemUrl, async () => {
+                const response = await got.get(itemUrl);
```

Comment:
```suggestion
                const response = await got(itemUrl);
```
Stick with one convention only
https://github.com/DIYgod/RSSHub/blob/7c47ac8c957ff8878151a09d759019ee4846afe8/lib/v2/tencent/news/author.js#L8